models/mentor.js

```diff
@@ -17,20 +17,6 @@ export default sequelize.define('mentors', {
   },
 }, {
   scopes: {
-    time(time) {
```

Comment:
Yeah. It should have been deleted when i changed shifts to be backed by Google calendar since the model doesn't exist anymore but apparently I missed it then.